example/lib/util/pair.dart

```diff
@@ -0,0 +1,9 @@
+class Pair<T, S> {
```

Comment:
We don't need this.

Reply:
I created issue to refactor this test case.